src/widgets/settingspages/AdvancedPage.cpp

```diff
@@ -70,9 +71,98 @@ AdvancedPage::AdvancedPage()
                          []() mutable {
                              getSettings()->cachePath = "";  //
                          });
-
+    }
         // Logs end
+
+
+        // Timeoutbuttons
+    {
```

Comment:
All other settings leverage the SignalVectorModel for collection settings so this seems really out of place

Reply:
true, will do a rework.